go.mod

```diff
@@ -5,7 +5,7 @@ go 1.17
 require (
 	github.com/Masterminds/sprig/v3 v3.2.2
 	github.com/aquasecurity/libbpfgo v0.3.0-libbpf-0.8.0
-	github.com/aquasecurity/tracee/types v0.0.0-20220704142452-9d0319c36c52
+	github.com/aquasecurity/tracee/types v0.0.0-20220725102908-2339d3e86117
```

Comment:
If any type changes are added until this pr is merged, they will be reverted and broken by this. I think there should still be a separate types PR.